.github/workflows/test.yml

```diff
@@ -18,6 +18,47 @@ jobs:
       matrix:
         march: [nehalem, armv7a, armv8a]
     steps:
-    - uses: seL4/ci-actions/camkes-vm@master
+    - name: Build images
+      uses: seL4/ci-actions/camkes-vm@master
       with:
         march: ${{ matrix.march }}
+    - name: Upload images
+      uses: actions/upload-artifact@v2
+      with:
+        name: images-${{ matrix.march }}
+        path: '*-images.tar.gz'
+
+  hw-test:
+    name: HW Test
+    runs-on: ubuntu-latest
```

Comment:
This one isn't going to work like this, unfortunately. We need to change the trigger to `pull_request_target` to access secrets and also adding labelling etc to the trigger for it to be called then. It then has to be merged first so that `pull_request_target` can become active.

Also, the camkes-vm tests didn't actually fail, it was the camkes-vm-examples tests that failed. I can have a look later at adding label triggers for both of them for PRs on the camkes-vm  repo (will take a bit, I'm in meetings the rest of the day).